airbyte-webapp/src/views/Connector/ServiceForm/components/Property/PropertyLabel.stories.tsx

```diff
@@ -0,0 +1,34 @@
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+
+import { Card, Input } from "components";
+
+import { PropertyLabel } from "./PropertyLabel";
+
+export default {
+  title: "Composed/PropertyLabel",
```

Comment:
Connector/PropertyLabel may be a better place for this.

I would also move the ServiceForm story from views to Connector/ServiceForm